crates/cribo/debug_resolver.py

```
@@ -0,0 +1,16 @@
+import sys
+import os
+
+# Test what the resolver should find
+test_dir = "tests/fixtures/stickytape_test_scripts/script_using_from_to_import_module"
```

Pylint (3.3.7)

[convention] Constant name "test_dir" doesn't conform to UPPER_CASE naming style

(C0103)